app/src/main/java/com/example/weatherandroidapp/views/WeatherGridFragment.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.weatherandroidapp.views
+
+import androidx.lifecycle.ViewModelProvider
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.weatherandroidapp.R
+import com.example.weatherandroidapp.viewmodels.WeatherGridViewModel
+
+class WeatherGridFragment : Fragment() {
+
+    private lateinit var viewModel: WeatherGridViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.weather_grid_fragment, container, false)
```

Comment:
Please use binding to inflate this view!

Reply:
Done